grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -85,6 +85,31 @@ class ProductAvailabilityService {
         }
     }
 
+    def refreshProductsAvailability(String locationId, List<String> productIds, List<String> binLocationIds, Boolean forceRefresh) {
+        if (locationId && productIds && !binLocationIds) {
+            refreshProductsAvailability(locationId, productIds, forceRefresh)
+            return
+        } else if (!locationId || !productIds) {
```

Comment:
If there is a `return` statement, there is no need to use `else`

Reply:
changed